scripts/run-api-tests.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/sh
 
 script="$(dirname $0)/../docker-compose-api-tests.yml"
-docker compose -f $script up --build --abort-on-container-exit --exit-code-from api-tests
+docker compose -p tests -f $script up --build -d
+
+TESTS_RUNNER_CONTAINER_ID=$(docker ps -aqf "name=^tests-api-tests-runner-1$")
+TEST_EXIT_CODE=`docker wait $TESTS_RUNNER_CONTAINER_ID`
+
+# Log tests results
+docker logs $TESTS_RUNNER_CONTAINER_ID
+
+# Clean
+docker-compose -p tests kill
+docker-compose -p tests rm -f
```

Comment:
@rkamysz  Why don't you simply run the tests in the foreground?
`docker compose -p tests -f $script up --build` (withoud `-d`)?
Then you don't have to wait & kill the containers, they will simply finish when they are done.

Reply:
The first problem was that it was "loud" - a lot of logs... I wanted to see only test results in the console, nothing else so that it would be easy to read - also in Actions stage /git pipeline.

The second thing is that in the case of CI, the script must return the code from the `api-tests` (to continue with following steps or break the pipeline) and in the above example, regardless of the test result, the script returned `ERRO [some_id] 0 or other number`.

If you know of an easier way to achieve this, write suggestions